### ISSUE TITLE
Snapshot manager

### ISSUE THREAD

**Issue by larmeh** [jigish/slate#203]
Is there a way to select and possibly merge snapshots easily? Because I'm thinking about writing one and wouldn't want to reinvent the wheel in case I've missed it.

**Comment by jigish** [comment]
Nope, not yet. Seems like it would be useful though so if you get the chance go for it!

**Comment by will-riley** [comment]
+100 on additional snapshot buffers. This would be an extremely easy way for non-leets (those who don't do .config files) to enjoy the heck out of this app. If all slate did was create/manage a list of snapshots, a grand pile of users would be swimming in awesome (yes I know that's not slates raison d'etre. But...).
